docs/modules/player-lifecycle/basic-player-creation.md

```diff
@@ -0,0 +1,94 @@
+<%(TOC)%>
+# Implementing basic player creation
+
+<%(Callout message="
+Before reading this document, make sure you have read the following documentation on:
+
+* [Player Lifecycle Feature Module]({{urlRoot}}/modules/player-lifecycle/overview)
+* [Creating entity templates]({{urlRoot}}/reference/concepts/entity-templates)
+* [Workers in the GDK]({{urlRoot}}/reference/concepts/worker)
+")%>
+
+By default, the module automatically sends a player creation request as soon as the client-worker instance connects to SpatialOS. This spawns a [SpatialOS entity]({{urlRoot}}/reference/glossary#spatialos-entity) representing a player.
+
+To do this, you need to:
+
+1. Set up your worker connector.
+1. Define the entity template for your player entities.
+1. Configure what entity template to use for player creation.
+
+## Set up your worker connector
```

Comment:
Would be nice to keep this consistent across every feature module documentation

Reply:
I tried to mimic the worker connector bit from the gameobject creation module docs

Reply:
I think its clearer if we don't have bullets and inlined code for the snippets, i.e. - 

---

You need to add the underlying systems to your worker. Open your [`WorkerConnector` implementations]({{urlRoot}}/reference/workflows/monobehaviour/creating-workers) and add one of the following lines to the `HandleWorkerConnectionEstablished` method.

**If this is a client-worker:**

```csharp
    TransformSynchronizationHelper.AddClientSystems(Worker.World);
```

**If this is a server-worker:**

```csharp
    TransformSynchronizationHelper.AddServerWorkers(Worker.World);
```

Reply:
That makes sense, and thanks for the neat example 

will update